API-1851:  Event API: precise the URL target location can not be localhost (#628)
akeneo/pim-api-docs: content/getting-started/quick-start-my-first-webhook-5x/step-1.md

Here, you will connect an event subscription to a fake endpoint and receive your first data. :rocket:

::: info

> For security reasons, localhost is blocked for event API target location and cannot be used as a fake endpoint. Use Mockbin or a comparable tool instead.

Reviewer: ```suggestion
For security reasons, local URLs are blocked for event API target location and cannot be used as a fake endpoint. Use Mockbin or a comparable tool instead.
```
